src/Microsoft.DotNet.Darc/DarcLib/Helpers/VersionDetailsParser.cs

```diff
@@ -193,4 +193,13 @@ private static XmlDocument GetXmlDocument(string fileContent)
 
         return document;
     }
+
+    public static string SerializeSourceDependency(SourceDependency sourceDependency)
+    {
+        return $"<{SourceElementName} " +
```

Comment:
it's not that big, but maybe we should use a string builder?

Reply:
I believe since this is a single extrapolated string and not something built dynamically in a loop, it gets lowered to quite efficient code.
Also I don't think this is a particularly hot area and string builder would make it look quite unreadable.

So I would actually say let's not?